src/main.ts

```diff
@@ -59,10 +63,16 @@ async function main() {
             type: 'string',
             describe: 'The file to translate (leave out for stdin)',
           })
+          .option('language', {
+            type: 'string',
+            describe: 'Language ID to transliterate to',
+            choices: Array.from(new Set(Object.values(TargetLanguage))),
+          })
           .option('python', {
             alias: 'p',
             boolean: true,
-            description: 'Translate snippets to Python',
+            deprecated: true,
+            description: 'Translate snippets to Python. Use --language python instead.',
```

Comment:
I also added the same option to markdown for consistency but let me know if I should remove this.